packages/shared/src/keys.ts

```diff
@@ -31,7 +31,7 @@ export function parsePublishableKey(key: string | undefined): PublishableKey | n
   if (!frontendApi.endsWith('$')) {
     return null;
   }
-
+  // TODO(@dimkl): validate packages/clerk-js/src/utils/instance.ts
```

Comment:
?

Reply:
A reminder to move `validateFrontendApi` to shared from `packages/clerk-js/src/utils/instance.ts` and use it in here to validate the decoded `frontendApi`.
I will remove this in a later PR